src/devtools/client/debugger/src/components/Editor/LineNumberTooltip.tsx

```diff
@@ -116,20 +117,15 @@ export default function LineNumberTooltip({
     if (lineNumber !== lastHoveredLineNumber.current) {
       lastHoveredLineNumber.current = lineNumber;
     }
-    if (prefsService.getBoolPref("devtools.features.codeHeatMaps")) {
-      setTimeout(() => {
-        if (lineNumber === lastHoveredLineNumber.current) {
+    setTimeout(() => {
+      if (lineNumber === lastHoveredLineNumber.current) {
```

Comment:
lifted these conditions up so that it was a bit more obvious what the feature flag was doing